privval/socket_listeners_test.go

```diff
@@ -104,33 +104,39 @@ func TestListenerTimeoutReadWrite(t *testing.T) {
 		// Note: this controls how long this test actually runs.
 		timeoutReadWrite = 10 * time.Millisecond
 	)
-	for _, tc := range listenerTestCases(t, timeoutAccept, timeoutReadWrite) {
-		go func(dialer SocketDialer) {
-			_, err := dialer()
+
+	for i := 0; i < 100; i++ {
```

Comment:
Do we need to repeat 100 times?

Reply:
![image](https://github.com/cometbft/cometbft/assets/7142025/fac4c6b5-c05c-40ed-a524-3b75da3eebc9)


(and I checked and it costs a few seconds)

Reply:
But the same could be said about all tests. Does that mean we have to repeat them 100 times too?

Reply:
I suppose your POV is also accurate.

It passes 100 rounds well.